src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dashboard/ProgressOverview.java

```diff
@@ -62,6 +62,7 @@ protected ProgressOverview(AbstractTank tank) {
      */
     protected final void setFillLevelProgressEnabled(boolean progressEnabled) {
         fillLevel.setProgressEnabled(progressEnabled);
+        tabFillLevel.setDisable(!progressEnabled);
```

Comment:
Also ich finde es in Ordnung die Tabs zu deaktivieren. Das deaktivieren der Charts finde ich nicht sichtbar genug. Was meinen die anderen dazu?

@ByteHamster Was meinst du damit? Wenn ich den Monitor ohne Simulation starte, dann sind die Tabs und Diagramme bei mir deaktiviert.